### PR TITLE
[Shows] Added a ShowView submodule

### PR DESCRIPTION
**Pull request by sarahscott**
Should I add Pods back to the gitignore?

### CODE REVIEW

**Comment by alloy** [comment]
No they should remain. I don't get why the image view component pod and its dependencies were removed. You ran 'pod install' ?

You don't actually need to run 'pod install' btw, because we're checking them in, it should just build.

**Comment by alloy** [comment]
I’m also thinking that we might need to reconsider how we name modules. I realise that I called the switch and tab views ‘SomethingSomethingView’, but I don’t think that’s actually the convention with React/Native. E.g. ‘Headline’ vs ‘HeadlineLabel’ and ‘Artist’ vs ‘ArtistView’ seems to be more the convention.

What do you think?

**Comment by alloy** [comment]
@craigspaeth Do you have any thoughts on that naming convention in the React world? ^^

**Comment by alloy** [comment]
@sarahscott Oohh, I totally get now why those Pods were removed for you, I had not yet pushed changes to the podspec, my bad.

In any case, with checked-in dependencies, you don’t need to run `pod install` _unless_ you make any changes to the `Podfile`.

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
```
Comment (sarahscott) [inline tsconfig.json]: How do you feel about this file?

**Comment by sarahscott** [comment]
Maybe we can merge this one for now and I can start on a new one for multiple show modules within the main show view?

File: tsconfig.json (hunk 1 above)
Comment (alloy) [inline tsconfig.json]: I have the feeling that this is for TypeScript, we shouldn’t need that yet, right?

**Comment by sarahscott** [comment]
I removed the tsconfig for now.

**Comment by alloy** [comment]
Ace 👍

**Pull request merged**